src/site/content/en/metrics/lcp/index.md

```diff
@@ -241,54 +249,24 @@ po.observe({type: 'largest-contentful-paint', buffered: true});
 // leaves the tab.
 addEventListener('visibilitychange', function fn() {
```

Comment:
Above, you can use startTime: `lcp = lastEntry.startTime;`

Reply:
Yeah, this is a bit weird (and others have commented on it too), but I do want to emphasize that both are available. I think it's worth reworking the example in a subsequent PR.